docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -0,0 +1,108 @@
+and is pronounced [/ɛmjuːaɪ/](http://ipa-reader.xyz/?text=%C9%9Bmju%CB%90a%C9%AA).
+
+It's shorter, it distinguishes ourselves, it's already how many people abbreviate us.
+More importantly, it allows for the release of products not directly coupled to Material Design, such as unstyled/headless version of the components, a brand new second design system as an alternative to MD, and more.
+
+### A new domain name
+
+A new name wouldn't be so official without a new domain.
+So, we have moved home from https://material-ui.com/ to https://mui.com/, a very easy-to-type (and find) 3 letters domain.
+
+### New logos
+
+We've tightened up the company's logo to match the new branding.
+
+It's basically the same, to keep it familiar.
+We're keeping the geometrical shape, to resonate with the _building blocks_ idea of the components.
+We're also reducing the emphasis on the smart 3D perspective, stepping away from the notion of elevation Material Design coined.
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/old-new.png" alt="" style="width: 302px; margin-bottom: 16px;" />
+
+We are introducing logos for each of our products:
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/products.png" alt="" style="width: 427px; margin-bottom: 16px;" />
+
+### New package names
+
+The package names have changed in v5, which is a **breaking change**, you can find more details in the release notes.
+The npm organization name has moved from @material-ui to [@mui](https://www.npmjs.com/org/mui).
+
+### A new website & documentation
+
+Lastly, to celebrate and start a new chapter of the company, together with v5.0.0 rolling out and new products in the making, we're introducing a brand new marketing website and documentation design.
+
+You'll find a completely different theme from Material Design, new typeface, colors, box-shadows, etc.
+All done using the sophisticated theming features.
+We want to position the company in a broader space, better explain and share what we do and who we are.
+
+<a href="/"><img loading="lazy" src="/static/blog/material-ui-is-now-mui/palette.png" alt="" style="width: 408px; margin-bottom: 16px;" /></a>
+
+<p class="blog-description">The color palette of our brand.</p>
+
+[Head to the new website](/).
+
+> Note that these changes are **not** impacting the design of the components released under the `@mui/*` npm packages.
+
+## The path ahead
+
+We have a lot of exciting stuff to come in the near future as we roll into this new chapter.
+We're grateful to have you together with us, supporting the library and making it grow bigger and bigger.
+If you spot any errors or inconsistencies, please warn us so we fix them.
+
+Material Design will continue to be supported with the same attention to detail as before.
+We hope to keep serving those who like the library for closely following MD while also leveraging the strong foundation built so far to offer new components, designs, and products.
```

Comment:
```suggestion
We hope to keep serving those who like the library for closely following MD, while also leveraging the strong foundation built so far to offer new components, designs, and products.
```

Reply:
"while also leveraging" 

Congratulations 😆

Reply:
It's a suggestion from someone you might know, but not me 😛

Reply:
Ah sorry, I think I have something with this word, I kinda like it, it doesn't exist in Portuguese, maybe that's why 🙄

Reply:
Oh, I miss read, I though Matt meant it was a correct use here, to use the foundation as a lever 😁